tests/gtest/avifrgbtoyuvtest.cc

```diff
@@ -411,6 +411,59 @@ TEST(RGBToYUVTest, AllMatrixCoefficients) {
   }
 }
 
+TEST(RGBToYUVTest, 8BitGrayToYUV420) {
+  // 2x2 8-bit image
+  const uint8_t gray[4] = {4, 3, 2, 1};
```

Comment:
All of the const arrays could be `static`

Reply:
Done.